src/clj/slf4j_timbre/factory.clj

```diff
@@ -0,0 +1,22 @@
+(ns slf4j-timbre.factory
+	(:gen-class
+		:name com.github.fzakaria.slf4j.timbre.TimbreLoggerFactory
+		:implements [org.slf4j.ILoggerFactory]
+		:state state
+		:init init)
+	(:require slf4j-timbre.adapter)
+	(:import com.github.fzakaria.slf4j.timbre.TimbreLoggerAdapter))
+
+(defn -init
+	[]
+	[[] (ref {})])
+
+(defn -getLogger
+	[this logger-name]
+	(dosync
```

Comment:
nitpick: Does dosync use locks under the hood ? It might be not equal to a non-locking concurrent hashmap.